components/raftstore/src/store/fsm/peer.rs

```diff
@@ -2681,9 +2681,13 @@ where
         if self.fsm.peer.is_leader() {
             return;
         }
-        let mut resp = ExtraMessage::default();
+        let mut resp: ExtraMessage = ExtraMessage::default();
         resp.set_type(ExtraMessageType::MsgAvailabilityResponse);
         resp.wait_data = self.fsm.peer.wait_data;
+        let report = resp.mut_availability_context();
```

Comment:
Can you add comments on how is the information used in TiFlash

Reply:
This information is used by v2. https://github.com/tikv/tikv/blob/c16cab2c29ba36af51826e9b7d4e8bd950e0c15e/components/raftstore-v2/src/operation/command/admin/merge/prepare.rs#L371